nx/include/switch/services/set.h

```diff
@@ -885,3 +1007,33 @@ Result setsysGetTouchScreenMode(SetSysTouchScreenMode *out);
  * @param[in] mode \ref SetSysTouchScreenMode
  */
 Result setsysSetTouchScreenMode(SetSysTouchScreenMode mode);
+
+/// Initialize setcal.
+Result setcalInitialize(void);
+
+/// Exit setcal.
+void setcalExit(void);
+
+/// Gets the Service object for the actual setcal service session.
+Service* setcalGetServiceSession(void);
+
+/// Gets EciDeviceCertificate of size 0x180.
```

Comment:
`Gets the \ref {full struct name}.`

Reply:
I have updated the comments to match that format.